### PR TITLE
Display the popovers in VM Template Details tab correctly

### PR DESCRIPTION
**Pull request by hstastna**
## 📝 Description
Display the content of the popovers for _Name_, _Namespace_, _Labels_, _Annotations_, _Owner_, _Created at_ correctly - with the additional empty line above the breadcrumbs in the popover's content.

**Before:**
Name popover:
name_before
Owner popover:
owner_before

**After:**
Name popover:
name_after
Owner popover:
owner_after

### CODE REVIEW

**Comment by hstastna** [comment]
@glekner @avivtur @metalice @pcbailey @yzamir @vojtechszocs please review

File: src/views/templates/details/tabs/details/components/Annotations.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ import {
   Popover,
 } from '@patternfly/react-core';
 
+import 'src/views/templates/details/tabs/details/TemplateDetailsPage.scss';
```
Comment (yaacov) [inline src/views/templates/details/tabs/details/components/Annotations.tsx:15]: the annotations component should not need this scss import,
this scss should be called from TemplateDetailsPage.ts
if you need it here, you need to look why, and adress the issue some other way

**Comment by yaacov** [comment]
/lgtm

@hstastna  hi, can you look why you need the scss imports and fix that issue ?

btw/ a hint that this import is problematic is that you needed to use import from non-trivial path (using full path is the same as using relative path, it's a way to import things that may not be planned to imported from your component) sometimes it is necessary to do import from relative paths, but it's a hint that something may be wrong

**Comment by hstastna** [comment]
@yaacov the issue is fixed, PTAL again. Thanks!

**Pull request merged**